packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -291,6 +294,10 @@ class WebhookService {
             return;
         }
 
+        if (!connectionIds || connectionIds.length === 0) {
+            await this.forwardHandler(environment_id, providerConfigKey, '', provider, payload, webhookOriginalHeaders);
```

Comment:
To avoid mistakes later

```suggestion
            await this.forwardHandler(environment_id, providerConfigKey, '', provider, payload, webhookOriginalHeaders);
            return;
```